task2/ConsoleHost.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task2
+{
+   
+    public static class ConsoleHost
+    {
+        const int FromCenterToLeft = 5;
+        const int FromCenterToTop = 5;
+        public static int left = Console.WindowWidth / 2 - FromCenterToLeft;
+        public static int top  = Console.WindowHeight / 2 - FromCenterToTop;
+       
+        public static void Run()
+        {
+            const string  ShapeName = "rectangle";
```

Comment:
Think to change this part to accept all types of Shapes. Imagine there will be 100 types of shape.